src/lib/components/Label/tests/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const labelDataTestId = 'label';
```

Comment:
You can put these directly in the test files, no need to centralize them. Alternatively, export them from `LabelTest.svelte` by adding a `module` script tag: https://svelte.dev/docs#component-format-script-context-module

Reply:
Done.